### DISCUSSION TITLE
Trying to get telerik working.

### DISCUSSION THREAD

**Discussion by skuflow** [oqtane/oqtane.framework#3043] [Q&A]
I have followed the steps in #690.  I've created the nuget package for the module and uploaded it, restarted the application, created a page and added my module but the result is this error with no further explanation:

An Unexpected Error Has Occurred In admin.telerikconcept, admin.telerikconcept.Client.Oqtane: Could not resolve signature of method __Blazor.admin.telerikconcept.Index.TypeInference:CreateTelerikGrid_0

**Comment by sbwalker** [comment]
@skuflow if you would like assistance troubleshooting your issue, it would be helpful if you shared all of the specific integration details you implemented in your module

**Comment by nicpitsch** [comment]
See my comment in #3037.
